### DISCUSSION TITLE
Restapi V1 Tags filter not working

### DISCUSSION THREAD

**Discussion by tedo-develop** [shlinkio/discussions#80] [Help wanted] [answered]
Hello Acelaya,

we have a confusing problem with endpoint rest/v1/short-urls.
We try to filter short-urls with two or more tags. Unfortunately we get the wrong results.
![tags_filter](https://user-images.githubusercontent.com/39614855/154906745-249023d2-5044-4d6c-b84a-da5978b138d3.png)

Some idea whats going wrong? In this example we need all short-urls with the tags **news AND zoom**. 
Do we use the wrong filter?

Thanks for your help.

**Comment by acelaya** [comment]
What Shlink version are you using?

**Reply by tedo-develop** [reply]
https://github.com/shlinkio/shlink-web-client/releases/tag/v3.5.1

**Reply by acelaya** [reply]
I mean the server.

**Reply by tedo-develop** [reply]
Sorry my bad, also lates version: https://github.com/shlinkio/shlink/releases/tag/v3.0.3

**Reply by acelaya** [reply]
Then I don't understand the use of v1 in the path, as shlink-web-client has been using v2 for a long time.

**Reply by acelaya** [reply]
I see the problem. You have to send the body in JSON format. Shlink 3 no longer supports form-data or url-encoded.

**Reply by tedo-develop** [reply]
Thank you! The last part sent me in the right direction to solve the issue.

**Comment by acelaya** [comment, marked as answer]
https://github.com/shlinkio/shlink/discussions/1388#discussioncomment-2219013
